### PR TITLE
connect messages_view method to html

### PR DESCRIPTION
**Pull request by Yarinh55**
fix issue #134 
Hi @lioramilbaum 
@morbitton and I work on it.
we fixed the structure of the connect_db method,
rendered the messages_view method to the messages_view.html and update the jinja according that.

### CODE REVIEW

**Comment by lmilbaum** [comment]
Please ask mor to add a comment

**Comment by morbitton** [comment]
@lioramilbaum 
We work together

File: app.py [hunk 1]
```diff
@@ -18,21 +17,14 @@ def definedlog(fileHandler):
     return logger
 
 
-def connect_db():
+def connect_db(host, user, password, database):
+    import mysql.connector
```
Comment (lmilbaum) [inline app.py]: import statement should be in the top of the file. Unless, there is a hidden reason which I am not aware of.

File: templates/messages_view.html [hunk 2]
```diff
@@ -24,43 +24,16 @@
 
         <h1>View Messages</h1>
 
-        <!-- Example to delete -->
 
-        <p> This is an example </p>
+               {% for row in view %}
```
Comment (lmilbaum) [inline templates/messages_view.html:28]: Is there an alignment issue here?

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: templates/messages_view.html (hunk 2 above)
Comment (Yarinh55) [reply templates/messages_view.html:28]: Hi @lioramilbaum 
Before we implemented the messages reading from the DB they created muck message to display.
Now we are reading the messages from the DB, so this code is redundant

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**